src/main/java/com/box/sdk/BoxUser.java

```diff
@@ -113,6 +114,41 @@ public static BoxUser getCurrentUser(BoxAPIConnection api) {
     }
 
     /**
+     * Returns an iterable containing all the enterprise users.
+     * @param  api the API connection to be used when retrieving the users.
+     * @return     an iterable containing all the enterprise users.
+     */
+    public static Iterable<BoxUser.Info> getAllEnterpriseUsers(final BoxAPIConnection api) {
+        return getAllEnterpriseUsers(api, null);
+    }
+
+    /**
+     * Returns an iterable containing all the enterprise users that matches the filter and specifies which child fields
+     * to retrieve from the API.
+     * @param  api        the API connection to be used when retrieving the users.
+     * @param  filterTerm used to filter the results to only users starting with this string in either the name or the
+     *                    login. Use null if don't want filter.
```

Comment:
Minor wording suggestion - I think the second sentence should be something like "Can be null to not filter the results."